source/schemas/transform.rst

```diff
@@ -50,3 +50,13 @@ Projections
    stsci.edu/asdf/0.1.0/transform/affine.rst
    stsci.edu/asdf/0.1.0/transform/rotate2d.rst
    stsci.edu/asdf/0.1.0/transform/tangent.rst
+
+Polynomials
+-----------
+
+.. toctree::
+   :maxdepth: 1
+
+   stsci.edu/asdf/0.1.0/transform/polynomial.rst
+   stsci.edu/asdf/0.1.0/transform/shift.rst
+   stsci.edu/asdf/0.1.0/transform/scale.rst
```

Comment:
Do `shift` and `scale` belong in the Polynomials section?  I see how they sort of are simple polynomials, but I'm not entirely sure one would look for them there.  Maybe under a new heading "simple transforms" or something?